book/learning_materials/75_control/40_trim_estimation.md

```diff
@@ -0,0 +1,53 @@
+# Online Wheel "Trim" Estimation  {#trim_estimation status=beta}
+
+In this section, a solution is proposed to specifically address the issue with a badly calibrated ``trim''. That is, generating the ability to dynamically adjust for an incorrect trim calibration online. Wheel "trim" refers to one wheel spinning faster than the other when they are both sent the same command, due to manufacturing discrepancies. This is perhaps a simpler approach to increasing controller robustness, as it takes advantage of a model. Consider a controller which outputs a commanded angular velocity, denoted $\omega^{\mathrm{com}}$, which is distinct form the actual vehicle angular velocity $\omega$. A commanded angular velocity leads to commanded right and left wheel velocities, denoted $\dot{\phi}_R^{\mathrm{com}}, \dot{\phi}_L^{\mathrm{com}}$ respectively. Now, suppose that these wheel velocities differ from actual wheel velocities $\dot{\phi}_R, \dot{\phi}_L$ by
```

Comment:
You've got the same issue here (`` + ''). Tbh, not sure which one it's supposed to be?